app/code/Magento/ProductVideo/Test/Mftf/Test/AdminValidateUrlOnGetVideoInformationTest.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminValidateUrlOnGetVideoInformationTest">
+        <annotations>
+            <stories value="Admin validates the url when getting video information"/>
+            <title value="Admin validates the url when getting video information"/>
+            <description value="Testing for a required video url when getting video information"/>
+            <severity value="CRITICAL"/>
+            <group value="ProductVideo"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+            <createData entity="ProductVideoYoutubeApiKeyConfig" stepKey="setStoreConfig"/>
+        </before>
+        <after>
+            <createData entity="DefaultProductVideoConfig" stepKey="setStoreDefaultConfig"/>
+            <actionGroup ref="logout" stepKey="logout"/>
+        </after>
+        <actionGroup ref="AdminOpenProductIndexPageActionGroup" stepKey="navigateToProductIndexPage"/>
+        <actionGroup ref="goToCreateProductPage" stepKey="goToCreateProduct">
+            <argument name="product" value="SimpleProduct"/>
+        </actionGroup>
+        <actionGroup ref="AdminOpenProductVideoModalActionGroup" stepKey="openAddProductVideoModal"/>
+        <actionGroup ref="AdminGetVideoInformationActionGroup" stepKey="clickOnGetVideoInformation"/>
+        <actionGroup ref="AdminAssertVideoValidationErrorActionGroup" stepKey="seeUrlValidationMessage">
+            <argument name="inputName" value="video_url"/>
+        </actionGroup>
+        <fillField selector="{{AdminProductNewVideoSection.videoUrlTextField}}"
```

Comment:
Please, create a separate action group (like AdminFillProductVideoUrlActionGroup) for filling this field and pass the value as argument.